news/scripts/news_cluster.py

```diff
@@ -0,0 +1,93 @@
+import argparse
+import datetime
+
+import pandas as pd
+from sklearn.cluster import AgglomerativeClustering
+from sklearn.feature_extraction.text import TfidfVectorizer
+
+from models.news import Cluster
+from mongo_news_db.db_connector import (delete_clusters, fetch_all,
+                                        insert_cluster)
+
+SECONDS_IN_DAY = 24 * 3600
+
+
+def main():
+    delete_clusters()
+
+    parser = script_args()
+    args = parser.parse_args()
+    pipeline = ArticleClustering(args.threshold)
+    pipeline.run()
+
+
+def script_args():
+    parser = argparse.ArgumentParser(
+        description='Articles fetcher for StockNewsAPI', formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument('--threshold', help='The maximum distance between articles that will be accepted in the cluster. Increasing this number decreases the size of clusters (Default=1.4)',
```

Comment:
The `ArgumentDefaultsHelpFormatter` will add that in for you.
```suggestion
    parser.add_argument('--threshold', help='The maximum distance between articles that will be accepted in the cluster. Increasing this number decreases the size of clusters',
```